src/Ui/Website/Extensions/StringExtensions.cs

```diff
@@ -35,6 +35,10 @@ public static string TryFormatBody(this string value, string? contentType)
                 return TryFormatJson(value);
             }
         }
+        else
+        {
+            return TryFormatJson(value);
+        }
 
         return value;
```

Comment:
Sorry for being picky but in this case this return is not needed, remove it please and it will be all good

Reply:
Not all code paths would return something then. Hope this solution is okay.

Reply:
That's not true. There is no other way to go out of the function than return TryFormatJson(value). And try formatJson sets on error the original not formatted body.